CHANGELOG.md

```diff
@@ -22,17 +22,21 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Scroll to top functionality on footer
 - Added `/modules/util/web-storage-proxy.js` utility module.
 - Added `/modules/util/js-loader.js` utility module.
+- Adds ESLint check for `@todo` jsdoc syntax.
```

Comment:
Yes, because when we go to tag next week's release (that PR should be opened on thur) we'll have to separate out what is and isn't included in the release.